app/styles/about.sass

```diff
@@ -1,36 +1,488 @@
+  line-height: 29px
+  
+  blockquote
+    border: none
+    
+    &:before 
+      font-family: "Monaco"
+      content: "\201C"
+      position: absolute
+      left: 0px
+      top: 20px
+      font-size: 40px
+      opacity: 0.5
+  
+  // Navbar
+
+  .navbar
+    background: white
+    margin-bottom: 0
 
+    a.navbar-brand
+      #logo-img
+        width: 210px
+        height: 65px
+        margin-right: 10px
 
-    img.img-thumbnail
-      background-color: transparent
+      .glyphicon-home
+        position: relative
+        top: 4px
+
+      color: $burgandy
       &:hover
-        background-color: rgba(200, 244, 255, 0.2)
-      float: left
-      width: 80px
-      height: 80px
-      margin: 0px 10px 22px 0px
+        color: white
+        background: $burgandy
+        
+    .navbar-toggle
+      color: black
+      margin: 30px 25px 0
 
-  .team_name
-    font-size: 20px
-    margin-top: 0
+    .nav > li > a
+      // TODO: Move this to bootstrap variables for navbars
+      font-weight: bold
+      font-family: $body-font
+      font-size: 16px
+      padding: 38px 15px 37px
+      color: $burgandy
+      text-shadow: 0 0 0
+
+      &:hover
+        background: $burgandy
+        color: white
+
+    #language-dropdown-wrapper
+      display: inline-block
+      padding: 30px 20px
+      .language-dropdown
+        width: 200px
+
+    .language-dropdown
+      width: 250px
+      
+    @media (max-width: $screen-sm-min)
+      .nav > li > a
+        padding: 10px 20px
+      #language-dropdown-wrapper
+        display: inline-block
+        padding: 10px 10px
+        .language-dropdown
+          width: 150px
+      
+    .img-circle
+      border: $gold 8px solid
+      width: 98px
+      height: 98px // Includes the border
+      
+    .user-level
+      position: absolute
+      top: 76px
+      right: 42px
+      color: $gold
+      text-shadow: 1px 1px black, -1px -1px 0 black, 1px -1px 0 black, -1px 1px 0 black
+      
+  // Buttons
+ 
+  .btn
+    border: none
+    border-radius: 5px
+    font-family: $body-font
+    font-weight: normal
+    background-image: none // overrides legacy buttons
+
+  .btn-primary, .btn-navy
+    background-color: $navy
+    color: white
+  
+  .btn-primary-alt, .btn-navy-alt
+    background-color: white
+    border: 1px solid $navy
+    color: $navy
+    
+  .btn-forest
+    background-color: $forest
+    color: white
+    
+  .btn-forest-alt
+    background-color: white
+    border: 1px solid $forest
+    color: $forest
+    
+  .btn-gold
+    background-color: $gold
+    color: white
+    
+  .btn-gold-alt
+    background-color: white
+    border: 1px solid $gold
+    color: $gold
+  
+  .btn-lg
+    font-size: 18px
+    padding: 13px 35px
+    
+  // Classes
+  
+  .text-navy
+    color: $navy
+    
+  .bg-navy
+    background-color: $navy
+    color: white
+    h1, h2, h3, h4, h5, h6, a
+      color: white
+    a.btn-primary-alt
+      color: $navy
+
+
+#about-view
+  .bold
```

Comment:
I was having weird spacing issues because I failed to think of just putting a <b> inside a <p> (rather than have a <b> by itself).

I think <b> is probably more appropriate than <strong> since <strong> is spoken louder on screen readers (but <b> just does the styling).